packages/hURL/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=hURL
```

Comment:
Please always put `pkgname` in lowercase.

Reply:
@Edu4rdSHL Someone else wrote a new app named hurl ( lower ). I can rename to something else if you prefer ?

Reply:
I didn't saw it in our repos, how's it conflicting?

Reply:
aur/hurl-bin 1.1.0-1 (+0 0.00) (Installed)
    Hurl, run and test HTTP requests

Binary name is "hurl"

Reply:
how about hurl-encoder ?

Reply:
Well, that's in AUR, we don't care about names in AUR packages, see https://github.com/BlackArch/blackarch/issues/2992 for conext.

Reply:
Huh thats fine. Will pork me hah. I'll adjust.

Reply:
adjusted / not committed yet